canary/consumer-java/src/main/java/com/amazon/kinesis/video/canary/consumer/RealTimeFrameProcessor.java

```diff
@@ -0,0 +1,72 @@
+package com.amazon.kinesis.video.canary.consumer;
+
+import com.amazonaws.kinesisvideo.parser.mkv.Frame;
+import com.amazonaws.kinesisvideo.parser.mkv.FrameProcessException;
+import com.amazonaws.kinesisvideo.parser.utilities.FragmentMetadata;
+import com.amazonaws.kinesisvideo.parser.utilities.FragmentMetadataVisitor;
+import com.amazonaws.kinesisvideo.parser.utilities.FrameVisitor;
+import com.amazonaws.kinesisvideo.parser.utilities.MkvTrackMetadata;
+import com.amazonaws.services.cloudwatch.model.StandardUnit;
+
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Optional;
+
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+public class RealTimeFrameProcessor extends WebrtcStorageCanaryConsumer implements FrameVisitor.FrameProcessor {
+
+    private boolean isFirstFrame = false;
+
+    public static RealTimeFrameProcessor create() {
+        return new RealTimeFrameProcessor();
+    }
+
+    private RealTimeFrameProcessor() {
+    }
+
+    @Override
+    public void process(Frame frame, MkvTrackMetadata trackMetadata, Optional<FragmentMetadata> fragmentMetadata,
+            Optional<FragmentMetadataVisitor.MkvTagProcessor> tagProcessor) throws FrameProcessException {
+        if (!isFirstFrame) {
+            try {
+                long currentTime = System.currentTimeMillis();
+                long canaryStartTime = super.mCanaryStartTime.getTime();
+
+                // TODO: Add this to contants file for now
+                String filePath = "../webrtc-c/firstFrameSentTimeStamp.txt";
```

Comment:
Lets use the env reading here. Details in C part of the PR. Also, it is advisable to create this file in the `pwd` (in our case the build directory). After the run completes, or after the file is read, you can just delete the file in Java code, or if not programatically, removeFile in Jenkins groovy script.